docs/source/en/model_doc/mms.mdx

```diff
@@ -156,3 +159,81 @@ processor.tokenizer.vocab.keys()
 ```
 
 to see all supported languages.
+
+To further improve performance from ASR models, language model decoding can be used. See the documentation [here](https://huggingface.co/facebook/mms-1b-all) for further details.  
+
+### **Speech Synthesis (TTS)**
+
+Individual TTS models are available for each of the 1100+ languages. The models and inference documentation can be found [here](https://huggingface.co/facebook/mms-tts).
+
+### **Language Identification (LID)**
+
+Different LID models are available based on the number of languages they can recognize - [126](https://huggingface.co/facebook/mms-lid-126), [256](https://huggingface.co/facebook/mms-lid-256), [512](https://huggingface.co/facebook/mms-lid-512), [1024](https://huggingface.co/facebook/mms-lid-1024), [2048](https://huggingface.co/facebook/mms-lid-2048), [4017](https://huggingface.co/facebook/mms-lid-4017). 
+
+#### Inference
+First, we install transformers and some other libraries
+```
+pip install torch accelerate torchaudio datasets
```

Comment:
Propagating the suggestions from https://github.com/facebookresearch/fairseq/pull/5186
```suggestion
pip install torch datasets[audio]
```

Reply:
Should we update the LID docs as well - https://huggingface.co/facebook/mms-lid-126  ?